Buttplug.Server.Managers.UWPBluetoothManager/UWPBluetoothDeviceInterface.cs

```diff
@@ -35,7 +35,7 @@ internal class UWPBluetoothDeviceInterface : ButtplugDeviceImpl
         [CanBeNull]
         private BluetoothLEDevice _bleDevice;
 
-        public override bool Connected => _bleDevice != null && _bleDevice.ConnectionStatus == BluetoothConnectionStatus.Connected;
+        public override bool Connected => _bleDevice != null;

```

Comment:
Corrective action for the case where _bleDevice.ConnectionStatus == BluetoothConnectionStatus.Connected occurs in some Vorze devices (Piston SA and UFOSA) even though the device is available.